gateway/CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 Changelog for `twilight-gateway`.
 
+## [0.4.2] - 2021-06-12
+
+### Additions
+
+There are 6 new `EventTypeFlags` ([#845], [#914] - [@7596ff]):
+
+- `EventTypeFlags::INTEGRATION_CREATE`
+- `EventTypeFlags::INTEGRATION_DELETE`
+- `EventTypeFlags::INTEGRATION_UPDATE`
+- `EventTypeFlags::STAGE_INSTANCE_CREATE`
+- `EventTypeFlags::STAGE_INSTANCE_DELETE`
+- `EventTypeFlags::STAGE_INSTANCE_UPDATE`
+
+### Changes
```

Comment:
I would call these Enhancements, Changes are for deprecations and breaking changes